docs/configuration.md

```diff
@@ -11,11 +11,14 @@ The release workflow needs a `release.yml` GitHub workflow in your repo, and spe
 ## Repo settings
 
 * Disable [branch protection **rules**](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)
-  on any branch the workflow will be pushing to (ie the default branch). Note that
-  PR https://github.com/guardian/gha-scala-library-release-workflow/pull/26 means that you _can_ use
-  branch protection **rulesets** to protect your default branch, so long as you allow your GitHub App
-  to bypass those restrictions.
-* **Guardian developers:** comply with the repository requirements of
+  on any branch the workflow will be pushing to (ie the default branch). Note that PR #26 means that you can use branch protection rulesets to protect your default branch, so long as you allow your GitHub App to bypass those restrictions.
+### Guardian developers 
+* Re-enable branch protections:
+  * Set the custom property `production_status` to `production` to apply  branch protection to the default branch via a ruleset (this allows the Scala release app to bypass branch protection).
+  * Create a separate 'Status checks' ruleset in your repo with the Branch protection property 'Require status checks to pass' -> 'Require branches to be up to date before merging' and add your repo's workflow name(s) to the 'Status checks that are required' list. For example, if your repo has a CI workflow with the name 'CI', then the setting would look like this:
```

Comment:
Someone unfamiliar with rulesets will probably assume a 'Status checks' ruleset means something specific, rather than the arbitrary(-ish) name we've assigned to the ruleset.

```suggestion
  * Create a second ruleset in your repo with the Branch protection property 'Require status checks to pass' -> 'Require branches to be up to date before merging'. Type your repo's workflow name(s) (not to be confused with the filename) into the 'Status checks that are required' box. For example, if your repo has a CI workflow with the name 'CI', then the setting would look like this:
```
The first sentence is a bit long so maybe we could break it up a bit? I've suggested a couple of other bits that I think would help with clarity